src/Lykke.Job.BlockchainCashinDetector.Core/Domain/CashinResult.cs

```diff
@@ -4,6 +4,7 @@ public enum CashinResult
     {
         Unknown,
         Success,
-        Failure
+        Failure,
+        OutdatedBalance
```

Comment:
Nothing about incomleted cashin.
May be it is Ok.

Reply:
Unknown == incompleted